bridge-ui/src/services/tokenService.ts

```diff
@@ -170,3 +171,58 @@ export async function fetchTokenPrices(
   const data = await response.json();
   return data;
 }
+
+export async function getTokenConfig(): Promise<NetworkTokens> {
+  const [mainnetTokens, sepoliaTokens] = await Promise.all([
+    getTokens(NetworkTypes.MAINNET),
+    getTokens(NetworkTypes.SEPOLIA),
+  ]);
+
+  const updatedTokensConfig = { ...defaultTokensConfig };
+
+  updatedTokensConfig.MAINNET = [
+    ...defaultTokensConfig.MAINNET,
+    ...(await Promise.all(
+      mainnetTokens.map(async (token: Token): Promise<TokenInfo> => {
+        const tokenType = token.symbol === USDC_TYPE ? TokenType.USDC : TokenType.ERC20;
+        try {
+          await fetch(token.logoURI);
+        } catch (error) {
+          token.logoURI = "/images/logo/noTokenLogo.svg";
+        }
+
+        return {
+          name: token.name,
+          symbol: token.symbol,
+          decimals: token.decimals,
+          type: tokenType,
+          L1: token?.extension?.rootAddress ?? null,
+          L2: token.address,
+          UNKNOWN: null,
+          image: token.logoURI,
+          isDefault: true,
+        };
+      }),
```

Comment:
the object construction is the same as that of Sepolia - you could build a generic object builder from token object and return that instead of two places doing exactly the same thing.